shared/wallets/wallets-and-details/index.js

```diff
@@ -3,15 +3,15 @@ import * as React from 'react'
 import {Box2, Reloadable} from '../../common-adapters'
 import WalletList from '../wallet-list/container'
 import {globalColors, styleSheetCreate} from '../../styles'
-import {isWaitingKey} from '../../constants/wallets'
+import {checkOnlineWaitingKey} from '../../constants/wallets'
 
 type Props = {
   children: React.Node,
   reload: () => void,
 }
 
 const Wallets = (props: Props) => (
-  <Reloadable waitingKeys={isWaitingKey} onReload={props.reload} reloadOnMount={true}>
+  <Reloadable waitingKeys={checkOnlineWaitingKey} onReload={props.reload} reloadOnMount={true}>
```

Comment:
The problem with using isWaitingKey is that errors get binned with their waiting key -- so even when you click reload, which only fires off the loadAccounts action, if any other action has an error, you'll still be on the Reloadable screen.